app/src/main/java/com/duckduckgo/widget/VoiceSearchWidgetConfigurator.kt

```diff
@@ -33,7 +33,7 @@ class VoiceSearchWidgetConfigurator @Inject constructor(
         remoteViews: RemoteViews,
         fromFavWidget: Boolean,
     ) {
-        if (voiceSearchAvailability.isVoiceSearchSupported) {
+        if (voiceSearchAvailability.isVoiceSearchAvailable) {
```

Comment:
Not in this specific part of code but seems like the widgets are not being updated  when remote config is updated.

Consider these steps:
1.) Install build
2.) Verify that icon is visible
3.) Add widget to home and verify icon is visible there.
4.) Update remote config to exclude device’s manufacturer
5.) Restart app
6- Verify that icon disappears from browser

Howere when I check the icon, it is not updated.